tests/scripts/all.sh

```diff
@@ -76,41 +76,14 @@
 # Notes for maintainers
```

Comment:
Moving code around conflicts with work on driver-only-ECC that we want to include in the upcoming release. Can we wait until after the code freeze?